pkgs/development/python-modules/sphinx-autobuild/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, livereload
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-autobuild";
+  version = "2020.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vbaf4vrxahylyp8zrlw5dx1d2faajp926c3pl5i1wlkp1yll62b";
+  };
+
+  # No tests included.
+  doCheck = false;
+
+  propagatedBuildInputs = [ sphinx  livereload ];
```

```suggestion
  propagatedBuildInputs = [ sphinx  livereload ];

  # No tests included.
  doCheck = false;
  pythonImportsCheck = [ "sphinx_autobuild" ];
```

I can't check the imports name because the repo is no longer there. I think this project might be dead and we maybe don't want to add it to nixpkgs.

---

Repo seems to have changed, see https://github.com/NixOS/nixpkgs/pull/109565/#discussion_r559096349

Updated in 4aaae200651ceb9d58c6881d6d8663528c0b20f8